src/main/webapp/script.js

```diff
@@ -559,3 +577,14 @@ function searchQuestion() {
     });
   }, false);
 })();
+
+/**
+ * Erases the query string from the url.
+ */
```

Comment:
In what contexts exactly is this used?

Reply:
Whenever a search is made (no matter the view) the user is redirected to index.html and the query string gets a pair of parameters. If the user clicks the logo to go from the search to the full forum, the query string stays there because no redirection was made. Therefore, this method erases the query string whenever this flow is followed.

Reply:
I see, so otherwise the query params would persist. SGTM, as long as you don't depend on any other query params to save other state of the application.

Reply:
That is right, and every time a search is performed, the query parameters will pile up.